Packs/UrlScan/Integrations/UrlScan/UrlScan.yml

```diff
@@ -40,6 +40,11 @@ configuration:
   name: is_public
   required: false
   type: 8
+- defaultvalue: 'true'
+  display: Create Relationships
+  name: relationships
```

Comment:
please change it to `create_relationships` because this word is heavily used and it is not comfortable to search the param.